esphome/components/e131/e131_addressable_light_effect.h

```diff
@@ -0,0 +1,66 @@
+#pragma once
+
+#include "esphome/core/component.h"
+#include "esphome/components/light/addressable_light_effect.h"
+
+#include "ESPAsyncE131.h"
+
+namespace esphome {
+namespace e131 {
+
+class E131Component;
+
+enum E131LightChannels {
+  E131_MONO = 1,
+  E131_RGB = 3,
+  E131_RGBW = 4
+};
+
+class E131AddressableLightEffect : public esphome::light::AddressableLightEffect {
+public:
+  E131AddressableLightEffect(const std::string &name);
+
+public:
+  void start() override;
+  void stop() override;
+  void apply(esphome::light::AddressableLight &it, const esphome::light::ESPColor &current_color) override;
```

Comment:
```suggestion
  void apply(light::AddressableLight &it, const light::ESPColor &current_color) override;
```

`esphome::` is not needed here as this is already in the esphome namespace.